clients/web/src/collections/UserCollection.js

```diff
@@ -8,6 +9,15 @@ var UserCollection = Collection.extend({
     // Override default sort field
     sortField: 'lastName',
     secondarySortField: 'firstName'
+}, {
+    getUsersCount: function () {
```

Comment:
How about naming this `getTotalCount` instead? We already know it's for users, since it's attached to the `UserCollection`.